ext/crypto/import_key.rs

```diff
@@ -548,10 +548,12 @@ fn import_key_ec_jwk(
         }
       };
 
+      let rng = ring::rand::SystemRandom::new();
```

Comment:
Should we maybe store it in a resource table to not incur creation cost on each op?